RPIPs/RPIP-59.md

```diff
@@ -68,7 +68,7 @@ ETH from the deposit pool SHALL be matched with validator deposits from queues a
 - If the beacon chain stake is invalid, the validator SHALL be scrubbed 
 - `stake` SHALL stake the remaining 31 ETH to the beacon chain to make a complete validator
 - If `stake` is not called within `time_before_dissolve` after the ETH was assigned, the validator SHALL be dissolved, returning the unstaked balance to the deposit pool
-  - If a validator is dissolved the bonded value SHALL be recoverable. This MAY require further action from the node operator. This MAY temporarily require additional ETH from the node operator
+  - If a validator is dissolved the bonded value SHALL be recoverable. This MAY require further action from the node operator. This MAY temporarily require additional ETH from the node operator.
```

Comment:
Not sure what convention (if any) we are using for periods, noticed elsewhere some sentences do/don't have periods. I didn't make a suggestion commit adding periods everywhere else since it seems unnecessary.

Reply:
it seems like, convention seems to be no period on bullet points. But I have some bullet points with multiple sentences (like here) and then it feels weird to have a period in the middle but not at the end.

Reply:
I feel like we should just change the convention to have periods on all bullets, but agree this is better than having periods in the middle and not end.

Reply:
Most style guides accept _either_ no periods on a bulleted list, or periods on every bullet. But also - I'm hitting the merge button. The relevant info is there and not confusing.